pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1376,7 +1376,8 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claim *v1.Persis
 	var pluginName string
 	provisionerName := storageClass.Provisioner
 	if plugin != nil {
-		if plugin.IsMigratedToCSI() {
+		// TODO(dyzz) Just temporary to test without dynamic provisioning
+		if plugin.IsMigratedToCSI() && false {
```

Comment:
temporary can be misleading without an actual roadmap (i.e. when is this going to be updated). if this TODO is a must, then a tracking issue should be in place.

Reply:
oversight, removed